crates/dprint/src/utils/glob_utils.rs

```diff
@@ -153,7 +166,7 @@ fn is_windows_absolute_path(file_path: &str) -> bool {
 
   // now check for the last slash
   let next_char = chars.next();
-  next_char == Some('/')
+  matches!(next_char, Some('/' | '\\'))
```

Comment:
is this tested against the real FS? If not, glob patterns are always posix?

Reply:
ah, this is a CLI pattern that is passed as arg

Reply:
This should actually do the file slash conversion ahead of time and not bother with this. Will fix.